scipy/signal/windows/_windows.py

```diff
@@ -1151,8 +1151,8 @@ def kaiser(M, beta, sym=True):
 
     The Kaiser was named for Jim Kaiser, who discovered a simple approximation
     to the DPSS window based on Bessel functions.
-    The Kaiser window is a very good approximation to the Digital Prolate
-    Spheroidal Sequence, or Slepian window, which is the transform which
+    The Kaiser window is a very good approximation to the discrete prolate
+    spheroidal sequence, or Slepian window, which is the transform which
```

Comment:
this one required me to check Wikipedia, but seems ok: https://en.wikipedia.org/wiki/Window_function

the rest of the changes seem straightforward typo fixes

Reply:
_cf_: aberrant acronym expansion

an emeritus professor at Stanford University